components/Market/utils.tsx

```
@@ -0,0 +1,202 @@
+}
+
+/**
+ * Formats the provided percent difference.
+ *
+ * @param percentDiff The percentage, expressed as a number where 100 means 100%.
+ * @returns The formatted percentage.
+ */
+export function formatPercentDiffSimple(percentDiff: number): string {
+  // Special case as we have a different tooltip for the equals case ("price is equal to...").
+  // We also don't display a tooltip when the difference is infinity.
+  if (percentDiff === 0 || !isFinite(percentDiff)) {
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)